PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionContentPanelConfigurable.java

```diff
@@ -105,6 +149,154 @@ protected void createUIComponents() {
         refreshClusterListAsync();
     }
 
+
+    @Nullable
+    public JPanel getMyWholePanel() {
+        return myWholePanel;
+    }
+
+    @Nullable
+    public String getLocalArtifactLibraryName() {
+        return localArtifactLibraryName;
+    }
+
+    @Nullable
+    public String getLocalArtifactPath() {
+        return localArtifactPath;
```

Comment:
We needn't to introduce this property.

Reply:
When user click the main class selection button, we should remove the previously added dependency unless user don't change the jar path. Therefore, we need `localArtifactPath` to know whether user change the jar path or not. Actually we can detect the jar path with `localArtifactLibraryName`, but wierd thing would happen. For example, user set the jar path to `C:\test\a.jar` but the jar path in the detected library is  `C:/test/a.jar`. I'm not sure if there is any other concerns, so I just record `localArtifactPath` for simplicity.